.expeditor/config.yml

```diff
@@ -89,8 +89,8 @@ changelog:
 merge_actions:
  - built_in:bump_version:
     ignore_labels:
-     - "Version: Skip Bump"
-     - "Expeditor: Skip All"
+     - "Expeditor: Skip All",
+     - "Expeditor: Skip Version Bump",
```

Comment:
I don't think you want trailing commas here.

Reply:
oops!